docs/docs.md

```diff
@@ -0,0 +1,628 @@
+# `/event/{eventKey}/info`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  webcasts: {
+    type: "twitch" | "youtube"
+    url: string
+  }
+  location: {
+    name: string
+    lat: number
+    lon: number
+  }
+  name: string
+  district?: string
+  week?: number
+  // UTC date
+  startDate: string
+  // UTC date
+  endDate: string
+  // Only if user is logged in
+  starred?: false
+}
+```
+
+# `/event/{eventKey}/match/{matchKey}/info`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  redAlliance: string[]
+  blueAlliance: string[]
+  // UTC date - match predicted time
+  time: string
+  // example: qm3
+  key: string
+  starred?: false
+}
+```
+
+# `/event/{eventKey}/match/{matchKey}/reports/{team}`
+
+## `PUT`
+
+### Request
+
+```ts
+type Data = {
+  teleop: (
+    | {
+        attempts: number
+        successes: number
+        statName: string
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+      }
+    | {
+        attempted: boolean
+        succeeded: boolean
+        statName: string
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+      }
+    | {
+        value: string
+        statName: string
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+      })[]
+  auto: (
+    | {
+        attempts: number
+        successes: number
+        statName: string
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+      }
+    | {
+        attempted: boolean
+        succeeded: boolean
+        statName: string
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+      }
+    | {
+        value: string
+        statName: string
+        // 2018orwil
+        event: string
+        // qm1
+        match: string
+      })[]
+}
+```
+
+
+### Response
+
+```ts
+type Data = {}
```

Comment:
I don't like this... the response body will be empty/null, not an empty object.

Reply:
Would it be empty or would it be `null`? empty is not valid JSON, but `null` is

Reply:
empty